pennylane/qinfo/transforms.py

```diff
@@ -258,17 +258,19 @@ def classical_fisher(qnode, argnums=0):
 
     Args:
         tape (:class:`.QNode` or qml.QuantumTape): A :class:`.QNode` or quantum tape that may have arbitrary return types.
+        argnums (Optional[int or List[int]]): Arguments to be differentiated in case interface ``jax`` is used.
 
-    Returns: func: The function that computes the classical fisher information matrix. This function accepts the same
-    signature as the :class:`.QNode`. If the signature contains one differentiable variable ``params``, the function
-    returns a matrix of size ``(len(params), len(params))``. For multiple differentiable arguments ``x, y, z``,
-    it returns a list of sizes ``[(len(x), len(x)), (len(y), len(y)), (len(z), len(z))]``.
+    Returns:
+        func: The function that computes the classical fisher information matrix. This function accepts the same
+        signature as the :class:`.QNode`. If the signature contains one differentiable variable ``params``, the function
+        returns a matrix of size ``(len(params), len(params))``. For multiple differentiable arguments ``x, y, z``,
+        it returns a list of sizes ``[(len(x), len(x)), (len(y), len(y)), (len(z), len(z))]``.
 
     .. warning::
 
         The ``classical_fisher()`` matrix is currently not differentiable.
 
-    .. seealso:: :func:`~.pennylane.metric_tensor`
+    .. seealso:: :func:`~.pennylane.metric_tensor`, :func:`~.pennylane.qinfo.quantum_fisher`
```

Comment:
Also this link does not work

Reply:
:func:`~.pennylane.qinfo.transforms.quantum_fisher`

Reply:
Try this one